google/cloud/forseti/services/model/importer/importer.py

```diff
@@ -457,15 +453,44 @@ def group_name(parent):
                 member_name=name)
             self.session.add(self.member_cache[member])
 
-        parent_group = group_name(parent)
+        parent_group = self.group_name(parent)
 
         if parent_group not in self.membership_map:
             self.membership_map[parent_group] = set()
 
         if member not in self.membership_map[parent_group]:
             self.membership_map[parent_group].add(member)
             self.membership_items.append(
-                dict(group_name=group_name(parent), members_name=member))
+                dict(group_name=self.group_name(parent), members_name=member))
+
+
+    def group_name(self, group):
+        """Create the type:name representation for a group.
+
+        Args:
+            group (object): group to create representation from.
+
+        Returns:
+            str: group:name representation of the group.
+        """
+
+        data = group.get_resource_data()
+        return 'group/{}'.format(data['email'].lower())
+
+    def _store_groups_settings(self, settings, group):
+        """Store gsuite settings.
+
+        Args:
+            settings (object): settings resource object.
+            group (object): group resource object.
+        """
+
+        settings_dict = settings.get_resource_data()
+        group_name = self.group_name(group)
```

Comment:
You can do group_name = self.group_name(settings) because settings includes the 'email' key.